src/components/card/card.scss

```diff
@@ -0,0 +1,196 @@
+
+@import "variables";
+@import "shadows";
+@import "default-theme";   // TODO: Remove this
+
+$md-card-default-padding: rem(2.400) !default;
+$md-card-mobile-padding: rem(2.400) rem(1.600) !default;
+$md-card-border-radius: 2px !default;
+$md-card-header-size: rem(4.000) !default;
+
+md-card {
+  display: block;
+  position:relative;
+  padding: $md-card-default-padding;
+  border-radius: $md-card-border-radius;
+  box-shadow: $md-shadow-bottom-z-1;
+  font-family: $font-family;
+  background: md-color($md-background, 50);       // temp until 0 works
+}
+
+%md_card_section_base {
```

Comment:
Could you give a brief comment on each of the placeholders describing what they represent?

The kinds of comments I have in mind are the like the comments in [this file from material 1](https://github.com/angular/material/blob/master/src/components/datepicker/datePicker.scss).